src/composables/useUtilities.ts

```diff
@@ -60,8 +60,8 @@ export default function useUtilities() {
   const clientSideSorter = (key: string, previousKey: string, sortOrder: string, items: []) => {
     let comparator = null
 
-    const numberComparator = (a: number, b: number) => {
-      if (a && b) {
+    const numberComparator = (a: number, b: number | string) => {
+      if (typeof b === 'number') {
```

Comment:
I'm not sure I completely understand the previous logic either; however, to be safe, should we do something like this:

```ts
if (a && typeof b === 'number') {
```

Reply:
This will make zeros fail the check, so we still cannot get the right order:

<img width="282" alt="image" src="https://user-images.githubusercontent.com/10095631/187821113-c2934b23-228b-406c-a806-d72503504710.png">

Reply:
Then what about something like this?
```ts
if (a !== undefined && typeof b === 'number') {
```

Reply:
Yeah, this looks fine but we already guarantee `a` is a number when calling `numberComparator `:
```js
switch (typeof newValA) {
  case 'number':
    return numberComparator(newValA, newValB)
```

Considering `typeof NaN` is also a `number`, maybe we should do something like this:
```js
if (!Number.isNaN(a) && !Number.isNaN(b)) {
```

Reply:
So I updated the code. Now we are sure `a` and `b` are both numbers and not `NaN`, so `a - b` won't give us any crazy result.